### PR TITLE
Implement tests for omniscdb-internal PR 5026

### PR DESCRIPTION
**Pull request by pearu**
Requires https://github.com/omnisci/omniscidb-internal/pull/5026

### CODE REVIEW

File: rbc/tests/test_omnisci_device_selection.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+def define(omnisci):
+
+    @omnisci('int32(int32)')
+    def rt_device_selection_udf_any(x):
+        # cannot determine which device is actually used
+        return any_device_code
+
+    @omnisci('int32(int32)', devices=['cpu'])
+    def rt_device_selection_udf_cpu(x):
+        return cpu_device_code
```
Comment (guilhermeleobas) [inline rbc/tests/test_omnisci_device_selection.py:135]: Add `devices` option to `RemoteOmnisci` class docstring.

Comment (pearu) [reply rbc/tests/test_omnisci_device_selection.py:135]: `devices` is actually a `Signature` argument it is documented there.

I think we need some kind of RBC/Omnisci tutorial document where we could document available features.

Comment (pearu) [reply rbc/tests/test_omnisci_device_selection.py:135]: Resolving this via https://github.com/xnd-project/rbc/issues/233

Comment (guilhermeleobas) [reply rbc/tests/test_omnisci_device_selection.py:135]: Maybe add this to one of the notebooks we have

**Pull request merged**